CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Note
 
 - Flag `autogenInternals` was removed, policy mutation has been removed.
+- Support upper case `Audit` and `Enforce` in `.spec.validationFailureAction` of the Kyverno policy, failure actions `audit` and `enforce` are deprecated and will be removed in `v1.11.0`.
```

Comment:
I set the removal of lowercase actions to `v1.11.0`. 

@vyankyGH is this aligned with our plan to drop v1 support?